Documentation/operations/upgrade.rst

```diff
@@ -356,6 +356,10 @@ Changed Metrics
   ``*_api_limiter_wait_duration_seconds`` metrics.
 * The ``cilium_bpf_map_pressure`` for policy maps is now exposed as a single
   label ``cilium_policy_*``, rather than a label per policy map of an endpoint.
+* The ``cilium_cidrgroup_policies`` has been renamed to
+  ``cilium_cidrgroups_referenced`` for better clarity.
+* The ``cilium_idrgroup_translation_time_stats_seconds`` has been disabled by
+  default.
```

Comment:
Would it be possible to explain why these changes occurred? Why was `cilium_cidrgroup_translation_time_stats_seconds` disabled by default? Why is `cilium_cidrgroup_referenced` better than `cilium_cidrgroup_policies`?

Reply:
This is explained in the first commit. I don't think it's really necessary to include the rationale also in the upgrade guide. The other notes in this section also don't explain why the changes were made. Ultimately, the main point of the guide is to inform users of changes. For more details, they can dig into the PRs, read changelog, etc.